voicevox_engine/synthesis_engine/synthesis_engine_espnet.py

```diff
@@ -246,9 +247,13 @@ def _synthesis_impl(self, query: AudioQuery, speaker_id: int):
         assert _speaker.text2speech is not None
         assert _speaker.token_id_converter is not None
 
+        # 話速用に書き換え
         with torch.no_grad():
-            tokens = query2tokens(query, _speaker.g2p)
+            tokens,speed = query2tokens(query, _speaker.g2p)
             ids = np.array(_speaker.token_id_converter.tokens2ids(tokens))
+            _speaker.text2speech.decode_conf.update({'alpha':2.1-speed,
+                              'noise_scale':0.333,
+                              'noise_scale_dur':0.333})
```

Comment:
`noise_scale`と`noise_scale_dur`を強制的に上書きすることになるので、`# fixme: 他エンジンへの対応`と追記してもらえると助かります。

ボイボエディタは話速の最大が2.0ですが、外部エディタはその限りではないので、チェックを入れておくと良さそうです。
また、`2.1-speed`となっていますが、これだとデフォルト（`speed=1.0`）の時に、alphaの値が1.1になるのが気になりました。
（推測ですが、恐らくalphaが0以下になるとエラーが出るため2.1にしているのかなと思いました。ご確認お願いします:bow:）
```suggestion
            speed = min(query.speedScale, 2.0)
            _speaker.text2speech.decode_conf.update({'alpha':2.1-speed,
                              'noise_scale':0.333,
                              'noise_scale_dur':0.333})  # fixme: 他エンジンへの対応
```